packages/dialtone-vue2/recipes/leftbar/callbox/callbox.vue

```diff
@@ -39,8 +39,10 @@
         <div class="d-recipe-callbox__content">
           <component
             :is="clickable ? 'button' : 'span'"
+            v-dt-tooltip="title"
             data-qa="dt-recipe-callbox__title"
             class="d-recipe-callbox__content-title"
+            tabindex="0"
             @click="handleClick"
           >
```

Comment:
Also added Tooltip (and made focusable) to the title since it can often be truncated.
<img width="325" alt="image" src="https://github.com/user-attachments/assets/9681b60f-a817-4bfd-be84-5437fa530a2f" />